index.js

```diff
@@ -0,0 +1,145 @@
+const execFile = require('child_process').execFile;
+
+const helmBinary = process.env.HELM_BIN;
+const args = process.argv;
+
+/**
+ * Prints the extension's usage
+ */
+function printUsage() {
+  const usage = "Converts helm's output to json format.\n"
+    + 'Works for commands: install, status\n'
+    + '\n'
+    + 'Example usage:\n'
+    + '  helm json install stable/rabbitmq\n'
+    + '  helm json status my-release-name\n';
+  process.stdout.write(usage);
+}
+
+// The expected arguments are:
+// ['node', 'js file', 'command', 'chart / release name']
+if (args.length < 4) {
+  printUsage();
+  process.exit(0);
+}
+
+/**
+ * Builds a json object from a string array of resources.
+ * format example:
+ * resourcesArr = ['==> v1beta1/StatefulSet \n
```

Comment:
resourcesArr should be renamed to resources
this comment should be formatted as a [jsdoc param](http://usejsdoc.org/tags-param.html) comment: @param {Array} resources

You can also add a jsdoc returns declaration